pkg/controllers/deprovisioning/controller.go

```diff
@@ -203,17 +213,17 @@ func (c *Controller) launchReplacementNodes(ctx context.Context, action Command)
 		return fmt.Errorf("cordoning nodes, %w", err)
 	}
 
-	nodeNames, err := c.provisioner.LaunchMachines(ctx, action.replacementNodes)
+	nodeNames, err := c.provisioner.LaunchMachines(ctx, action.replacementNodes, provisioning.WithReason(metrics.DeprovisioningReason))
```

Comment:
```suggestion
	nodeNames, err := c.provisioner.LaunchMachines(ctx, action.replacementNodes, provisioning.WithReason(reason))
```

Can we pass the reason through `launchReplacementNodes` so we can correlate with the termination reason?